clouddriver-kubernetes/src/main/groovy/com/netflix/spinnaker/clouddriver/kubernetes/v2/security/KubernetesV2Credentials.java

```diff
@@ -332,7 +336,11 @@ public KubernetesManifest get(KubernetesKind kind, String namespace, String name
   }
 
   public List<KubernetesManifest> list(List<KubernetesKind> kinds, String namespace) {
-    return runAndRecordMetrics("list", kinds, namespace, () -> jobExecutor.list(this, kinds, namespace));
+    if (kinds.isEmpty()) {
+      return new ArrayList<>();
+    } {
```

Comment:
why the extra block here? not criticizing, just curious, esp if this is intentional. as in, I would have either returned without the block or added an else.